### ISSUE TITLE
Train a model to use interactive function to refine the pretrained model.

### ISSUE THREAD

**Issue by FJGEODEV** [hkchengrex/MiVOS#46]
Hi there, thanks for your great work. 

I am wondering what if I want to use your template to train a model to implement this "interactive" template to refine the pretrained model, how can I do?

For example, I have the existing video and ground truth. I would like to train a model using those available data. Looks like I can use those to train a propagation model, how about fusion model and mask propagation model?

Very appreciated!

**Comment by hkchengrex** [comment]
The code for training the propagation module and the interaction module is in two different repo as listed in the readme file. If you want to finetune the modules you can use them.

**Comment by hkchengrex** [comment]
See https://github.com/hkchengrex/XMem/issues/51
